### PR TITLE
Scrum-19/MailMessage

### PR DESCRIPTION
**Pull request by RomaB-5**

### CODE REVIEW

File: libs/MailMessage/include/MailMessageBuilder.h [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        MailMessage Build();
+
+    private:
+        MailAddress m_from;
+        std::vector<MailAddress> m_to = {};
+        std::vector<MailAddress> m_cc = {};
+        std::vector<MailAddress> m_bcc = {};
+        std::string m_subject;
+        std::string m_body;
+        std::vector<MailAttachment> attachments;
```
Comment (vsh51) [inline libs/MailMessage/include/MailMessageBuilder.h]: Think of renaming class member variable (attachments) so that it meets code conventions

**Review by vsh51 — COMMENTED** [review]
The code is well-structured and easy to read. The organization and clarity of it make it easy to understand. Great job was done in ensuring readability and modularity. Just one tiny issue troubled me

File: libs/MailMessage/include/MailMessageBuilder.h [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+namespace ISXMM
+{
+    class MailMessageBuilder
+    {
+    public:
+        MailMessageBuilder& SetFrom(const std::string& email, const std::string& name = "");
+        MailMessageBuilder& AddTo(const std::string& email, const std::string& name = "");
+        MailMessageBuilder& AddCc(const std::string& email, const std::string& name = "");
+        MailMessageBuilder& AddBcc(const std::string& email, const std::string& name = "");
+        MailMessageBuilder& SetSubject(const std::string& subject);
```
Comment (YehorFur) [inline libs/MailMessage/include/MailMessageBuilder.h]: Setter should be prefixed with set_ and match the name of the variable

File: libs/MailMessage/include/MailMessageFormatter.h [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    class MailMessageFormatter
+    {
+    public:
+        inline static std::string boundary = "NEXT_PART_My4xNDE1OTI2NTM1";
+        static std::string MailFrom(const MailAddress& from);
+        static std::string MailTo(const std::vector<MailAddress>& to);
+        static std::string MailCc(const std::vector<MailAddress>& cc);
+        static std::string MailHeaders(const MailMessage& message);
+        static std::string MailBody(const MailMessage& message);
+        static std::string MailAttachmentHeaders(const MailAttachment& attachment, const std::string& filetype = "application/octet-stream");
```
Comment (YehorFur) [inline libs/MailMessage/include/MailMessageFormatter.h]: Static members should be prfixed with s_

**Review by YehorFur — COMMENTED** [review]
The code looks good and clean to me, but there are a couple of minor naming issues.

**Review by vsh51 — COMMENTED** [review]
The only change suggested: namespace content should not be indented

**Review by vsh51 — APPROVED** [review]

**Pull request closed without merging**